js/screens/AboutUs/AboutUs.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Text, ScrollView } from 'react-native';
+import styles from './styles';
+
+const AboutUs = () => {
+  return (
+    <ScrollView style={styles.container}>
+      <Text style={styles.header}>Our Mission</Text>
+      <Text style={styles.desc}>
```

Comment:
your font size does not match designer's vision. Please make the required changes.